var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -0,0 +1,112 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Elemental(CMakePackage):
+    """Elemental: Distributed-memory dense and sparse-direct linear algebra and optimization library."""
+
+    homepage = "http://libelemental.org"
+    url      = "https://github.com/elemental/Elemental/archive/v0.87.6.tar.gz"
+
+    version('0.87.6', '9fd29783d45b0a0e27c0df85f548abe9')
+
+    variant('debug', default=False, 
+            description='Builds a debug version of the libraries')
+    variant('shared', default=True, 
+            description='Enables the build of shared libraries')
+    variant('hybrid', default=True, 
+            description='Elemental: make use of OpenMP within MPI packing/unpacking')
+    variant('c_interface', default=False, 
+            description='Elemental: build C interface')
+    variant('python_package', default=False, 
+            description='Elemental: install Python interface')
+    variant('disable_parmetis', default=True, 
```

Comment:
1. Normally, we would make the `c_interface` default to True and `python_package` default to False.  The idea is that variants that add dependencies should default to False.

2. Please rename `python_package` to `python`, since this is standard.  And consider renaming `c_interface` to `c`.